### ISSUE TITLE
Various links in wiki are outdated

### ISSUE THREAD

**Issue by winstonrc** [MiSTer-devel/Main_MiSTer#757]
While setting up my MiSTer, I came across a number of links that seem to reference an older wiki url,  `MiSTer-devel/Main_MiSTer` instead of `MiSTer-devel/Wiki_MiSTer`, which breaks them.

Examples:
- `Upgraded power supply` bullet point in the [How to start with MiSTer requirements](https://github.com/MiSTer-devel/Wiki_MiSTer/wiki/How-to-start-with-MiSTer#requirements) section
- `HDMI-to-VGA converter picture` in the [Direct Video](https://github.com/MiSTer-devel/Wiki_MiSTer/wiki/Direct-Video#direct-video) section

To help rectify this, may I be granted access to edit the wiki?

**Comment by winstonrc** [comment]
Similarly, the picture linked [here](https://github.com/MiSTer-devel/Wiki_MiSTer/wiki/Direct-Video#direct-video) points to the incorrect wiki link and appears broken.

**Comment by sorgelig** [comment]
Ok. I've invited you.

**Comment by winstonrc** [comment]
I believe I have successfully updated every erroneous link. I tried to be mindful of not accidentally updating links that were intentionally pointing to the `Main_MiSTer` repo and only updating Wiki related links.

I didn't manually check that every link now properly works, but the handful that I randomly spot checked (including images and files), are all successfully pointing to the correct location now.
